jsonschema2kotlin-cli/src/main/kotlin/StringFormattingInterceptors.kt

```diff
@@ -22,13 +22,13 @@ class StringPropertyNameInterceptor(private val format: String) : PropertyInterc
     override fun intercept(schema: Schema, jsonPropertyName: String, specs: Pair<ParameterSpec, PropertySpec>): Pair<ParameterSpec, PropertySpec> {
         val newName = specs.first.name
             .let {
-                it[0].toUpperCase() + it.substring(1)
+                it.replaceFirstChar(Char::uppercaseChar)
```

Comment:
Might be nice to have high level documentation about what these interceptors are doing